website/content/docs/sync/index.mdx

```diff
@@ -127,6 +127,24 @@ Granularity can be updated. The new granularity only affects secrets newly assoc
 not modify the previously synced secrets. It is possible to update an association to force a recreate operation.
 The secret synced with the old granularity will be deleted and new secrets will be synced according to the new granularity.
 
+
+## Reconciliation
+
+Vault Secrets Sync is designed to automatically recover from transient failures
+in two ways: operation retries and reconciliation scans.
+
+Operation retries happen when a sync operation fails. Vault automatically
+retries the operation with exponential backoff. Operation retries help in
```

Comment:
Should we give general numbers? Like retries depend on the destination type but are usually in the range of 3 to 5 retries over 20 seconds at most. Something like that?

Reply:
I also think this could be subject to change and a bit low level for the user. Maybe in the future, we want this to be configurable as well.